carta/image.py

```diff
@@ -57,6 +57,8 @@ def new(cls, session, path, hdu, append):
             The HDU to open.
         append : boolean
             Whether the image should be appended. By default it is not, and all other open images are closed.
+        complex : None or string
+            Complex value to open a comlex-valued image. The valid options are ``AMPLITUDE``, ``PHASE``, ``REAL`` and ``IMAGINARY``. Default is ``AMPLITUDE``.
```

Comment:
@confluence, Done in 8b8c752.
As for the default value of `complex`, there are actually two kinds of default. One is that if the image is not a complex image, it is of course opened without any problem. The second is that if the image is a complex image, then we open it without `complex` argument set, the default image that opened is **AMPLITUDE**. I don't know how to describe that in short in the description, do you have any idea about it?

Reply:
I am not sure about 1., will take a look at it.
As for 2, I have just tested it, it does cause an error. Since the `complex` is applied to `file_name`, it's not able to look for the file name with `complex` attached to it.